eb

```diff
@@ -159,22 +160,45 @@ for argument in "$@"; do
 				letter=${letter,} # lowercase first letter
 				name=${argument%%-*}
 				# some names contain - (e.g. HOOMD-blue) so we need pathname expansion
-				ec=$(find $EASYBUILD_ROOT/easyconfigs/$letter/$name* -name $argument 2>/dev/null)
 				custom_ec=$(find $EASYBUILD_ROOT/site-packages/custom-easyconfigs/easybuild/easyconfigs/$letter/$name* -name $argument 2>/dev/null)
-				if [[ -n "$ec" && -f $ec && -z "$custom_ec" ]]; then
-					NEW_ARGS+=("$ec")
-				else    # if no corresponding easyconfig was found in easyconfigs folder, we get it found normally
-					NEW_ARGS+=("$argument")
+				if [[ -n "$custom_ec" && -f "$custom_ec" ]]; then
+					# custom easyconfigs have priority
+					ec="$custom_ec"
+				else
+					ec=$(find $EASYBUILD_ROOT/easyconfigs/$letter/$name* -name $argument 2>/dev/null)
+					if [[ -z "$ec" || ! -f "$ec" ]]; then
+						echo "Easyconfig $argument not found"
+						exit 1
+					fi
 				fi
+				NEW_ARGS+=("$ec")
 			fi
 		else
 			NEW_ARGS+=("$argument")
 		fi
+	done
+	if [[ -z "$ec" ]]; then
+		echo "No easyconfig specified, exiting"
+		exit 1
 	fi
-done
-
-if [[ "$CURRENT_USER" == "ebuser" ]]; then
-	exec $EB --configfiles=$LOCAL_EASYBUILD_CONFIGFILES "${NEW_ARGS[@]}"
+	cd /cvmfs/local/var/log
+	$EB --fetch --configfiles=$LOCAL_EASYBUILD_CONFIGFILES "${NEW_ARGS[@]}"
+	export RSNT_ARCH=avx2
+	$EB --job --configfiles=$LOCAL_EASYBUILD_CONFIGFILES "${NEW_ARGS[@]}"
+python <<EOF
+import sys
+from easybuild.framework.easyconfig.parser import EasyConfigParser
+ecp = EasyConfigParser("$ec")
+name = ecp.get_config_dict()['toolchain']['name']
+sys.exit(name == 'system' or name == 'dummy')
```

Comment:
Aren't the rules different for `StdEnv/2020` ? Again, we don't often build for it nowadays, but should we forbid it ?

Reply:
The rules are only different for `StdEnv/2016.4` and `2018.3`, where `GCCcore` is shared between architectures. I did not consider those though, since they are obsolete

Reply:
ok